daffodil-cli/src/it/scala/org/apache/daffodil/parsing/TestCLIParsing.scala

```diff
@@ -206,27 +206,19 @@ class TestCLIparsing {
   }
 
   @Test def test_1593_CLI_Parsing_MultifileSchema_noGlobalElem(): Unit = {
-    val tmp_filename: String = (System.currentTimeMillis / 1000).toString()
-    val file = new File(tmp_filename)
-    val schemaFile = Util.daffodilPath("daffodil-test/src/test/resources/org/apache/daffodil/section06/entities/TopLevel.dfdl.xsd")
```

Comment:
This schema has global elements and doesn't actaully test anything. I think this was originally meant to be a stub test or something. Modified it to test what the name implies it should test.